test/gtest/uct/v2/test_uct_query.cc

```diff
@@ -4,11 +4,9 @@
  * See file LICENSE for terms.
  */
 
-extern "C" {
```

Comment:
this is OK, but the fix should be to add this as the first include:
`#include <common/test.h>`

(to define __STDC_LIMIT_MACROS)